src/components/TimeseriesSearch/SelectedItems.tsx

```diff
@@ -0,0 +1,55 @@
+import { Tag } from 'antd';
+import React from 'react';
+import styled from 'styled-components';
+import { getColor } from './../../utils/colors';
+
+const Wrapper = styled.div`
+  flex-directoin: row;
+  margin-bottom: 8px;
+`;
+
+export interface Item {
+  id: number;
+  name: string;
+}
+
+export interface SelectedTimeseriesProps {
+  selectedItems: Item[];
+  onItemClose: (items: Item[]) => void;
+}
+
+export class SelectedItems extends React.Component<SelectedTimeseriesProps> {
+  static defaultProps = {
+    selectedItems: [],
+    onItemClose: null,
+  };
+
+  onClose = (item: Item) => {
+    const newItems = [...this.props.selectedItems].filter(
```

Comment:
For what you're creating new array for filtering it instead of using it directly from props?

Reply:
No reason at all. Fixed now